org.contextmapper.dsl/src/org/contextmapper/dsl/ContextMappingDSL.xtext

```diff
@@ -369,18 +375,26 @@ enum KnowledgeLevel :
   META="META" | CONCRETE="CONCRETE"
 ;
 
-enum Volatility : 
+enum Volatility :
 	UNDEFINED | NORMAL | RARELY | OFTEN
 ;
 
-enum Criticality : 
+enum Criticality :
 	UNDEFINED | NORMAL | HIGH | LOW
 ;
 
-enum Similarity : 
+enum Similarity :
 	UNDEFINED | NORMAL | HUGE | TINY
 ;
 
+enum BusinessModel :
```

Comment:
What's you opinion on those two new attributes on the bounded contexts @socadk? Any additions?

Reply:
I pulled the enum values for Business Model from "DDD Crew":
- https://github.com/ddd-crew/bounded-context-canvas#strategic-classification
- https://github.com/ddd-crew/bounded-context-canvas/blob/master/tools/html-version/bounded-context-canvas-template.html#L51

Evolution is based on Wardley maps.

Related: https://github.com/ddd-crew/core-domain-charts

Reply:
@boxleytw @stefan-ka Thanks for the proposal and the PR, supporting the canvas & Wardley concepts makes sense and is a useful addition (assuming that eventually these new language featured are picked up by the various generators, puml and so on). But are the enum values [mutually exclusive and collectively exhaustive (MECE)](https://en.wikipedia.org/wiki/MECE_principle)? I can think of more business models. If MECE principle is not met, it wold be great to add a free text STRING option to the grammar.

Reply:
I agree. Free text string as a fallback to define other values makes sense :+1:

Reply:
Thank you for feedback.
I'll update the PR today or tomorrow -- and update PRs in other CM repos to match.

Reply:
@socadk Would you also recommend that "Evolution" be free text?

Reply:
Actually ... I pushed this change today.

Reply:
I guess it should be fine like that @binkley. Sorry for the bad response time.